examples/001_Data_Ingress.ipynb

```diff
@@ -0,0 +1,664 @@
+    "print(f\"Existing Sensor Types: {auth_backend_call('get', '/sensor/list-sensor-types', token=access_token).json()}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Add a Sensor \n",
+    "\n",
+    "A sensor refers to the indivudal instance of a sensor or data source. It is associated with a sensor type. \n",
+    "Information required in the sensor payload the type name, a unique identifier, name of sensor and any notes. \n",
+    "\n",
+    "Remember, documentation of all endpoints can be found at `localhost:5000/docs`. \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 24,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5000\n",
+      "DEBUG:urllib3.connectionpool:http://localhost:5000 \"POST /sensor/list-sensors HTTP/1.1\" 200 2\n",
+      "DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5000\n",
+      "DEBUG:urllib3.connectionpool:http://localhost:5000 \"POST /sensor/insert-sensor HTTP/1.1\" 201 28\n",
+      "DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): localhost:5000\n",
+      "DEBUG:urllib3.connectionpool:http://localhost:5000 \"POST /sensor/list-sensors HTTP/1.1\" 200 169\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Existing Sensors: []\n",
+      "Existing Sensors: [{'unique_identifier': 'Tomorrow.io Weather Forecast - Location 1', 'name': 'Tomorrow.io Weather Forecast', 'notes': 'Weather forecast data from Tomorrow.io for Location 1'}]\n"
+     ]
+    }
+   ],
+   "source": [
+    "SENSOR = {\n",
+    "    \"unique_identifier\": \"Tomorrow.io Weather Forecast - Location 1\",\n",
+    "    \"type_name\": \"Weather: Tomorrow.io\",\n",
+    "    \"name\": \"Tomorrow.io Weather Forecast\",\n",
+    "    \"notes\": \"Weather forecast data from Tomorrow.io for Location 1\"\n",
+    "}\n",
+    "\n",
+    "# List Existing Sensors\n",
+    "print(f\"Existing Sensors: {auth_backend_call('post', '/sensor/list-sensors', payload={'type_name': SENSOR_TYPE['name']},  token=access_token).json()}\")\n",
+    "\n",
+    "# Add a Sensor\n",
+    "auth_backend_call('post', '/sensor/insert-sensor', payload=SENSOR, token=access_token)\n",
+    "\n",
+    "# List Existing Sensors again\n",
+    "print(f\"Existing Sensors: {auth_backend_call('post', '/sensor/list-sensors', payload={'type_name': SENSOR_TYPE['name']}, token=access_token).json()}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Add Sensor Readings\n",
+    "\n",
+    "Now we need to add the timeseries data. This data needs to be associated with a sensor's unique identifier. We call a sensor reading a measure. A sensor can have multiple measures. We have already defined these measures in the sensor type. \n",
```

Comment:
> We call a sensor reading a measure.

I would rephrase this somehow. A measure if e.g. "temperature in New York", a reading is "the temperature in New York at 8am this morning was 280 K".